### PR TITLE
Add Fcarrascosa's phpRiotApi

### PR DESCRIPTION
**Pull request by fcarrascosa**

### CODE REVIEW

**Comment by WxWatch** [comment]
Your `links` property doesn't pass validation. It needs to be an array like this:
```
{
    "owner": "WxWatch",
    "repo": "LoL-Fake-Java-Lib",
    "description": "This is a fake Java library for the Riot Games API",
    "language": "Java",
    "links": [
        {
            "name": "Documentation",
            "url": "https://github.com/WxWatch"
        }
    ],
    "metadata": {
        "v3-supported": true
    }
}
```

**Comment by fcarrascosa** [comment]
Thanks for your reply, I just fixed that!

File: libraries/php/phpRiotApi.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "owner": "fcarrascosa",
+  "repo": "phpRiotApi",
+  "language": "php",
+  "links": [
+    {
+      "name": "The project on gitHub",
+      "url": "https://github.com/fcarrascosa/phpRiotApi",
```
Comment (WxWatch) [inline libraries/php/phpRiotApi.json]: Let's go ahead and removing this trailing comma, for consistency and then we should be good to merge in 😄

**Comment by fcarrascosa** [comment]
Alright! Done again :P sorry for the inconveniences!! ;)

**Comment by WxWatch** [comment]
Looks good!

**Pull request merged**